migrations/20170410094328_weighted_wilsons.js

```diff
@@ -0,0 +1,34 @@
+// http://www.evanmiller.org/how-not-to-sort-by-average-rating.html
+exports.up = function(knex, Promise) {
+  return knex.schema.raw(`
+      CREATE FUNCTION wilsons(ups numeric, downs numeric) RETURNS numeric LANGUAGE plpgsql IMMUTABLE AS $$
```

Comment:
It wont since the function parameters are different.
wilsons(numeric, numeric) vs wilsons(int, int)